src/main/java/me/crespel/runtastic/model/ImagesMetaData.java

```diff
@@ -0,0 +1,46 @@
+package me.crespel.runtastic.model;
+
+import java.math.BigDecimal;
+import java.util.Date;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+import lombok.Data;
+
+/**
+ * Runtastic model for images meta data (\Photos\Images-meta-data).
+ * @author Christian IMFELD (imfeldc@gmail.com)
+ */
+@Data
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ImagesMetaData implements Comparable<ImagesMetaData> {
+
+    private Date createdAt;
```

Comment:
Other Date fields in models have a `@JsonFormat`, consider adding one here as well.

Reply:
I don't know what kind of format to add; as this date field is "just" a number in the file. See exampe below:
"created_at":1337957784000
How would the json format look like?

Reply:
You can use a @JsonFormat with a shape = JsonFormat.Shape.NUMBER, and no pattern, I think.

Reply:
Thanks, I added now:
	@JsonFormat(shape = Shape.NUMBER)
    private Date createdAt;

But it was even working with:
	@JsonFormat(shape = Shape.STRING, pattern = "yyyy-MM-dd HH:mm:ss Z")